backend/src/test/java/org/cryptomator/hub/api/UsersResourceTest.java

```diff
@@ -70,6 +69,20 @@ public void testGetAll() {
 					.body("id", hasItems("user1", "user2"));
 		}
 
+		@Test
+		@DisplayName("POST /users/me/access-tokens returns 200")
+		public void testPostAccessTokens() {
+			var body = """
+					{
+						"7E57C0DE-0000-4000-8000-000100001111": "jwe.jwe.jwe.vault1.user1",
+						"7E57C0DE-0000-4000-8000-BADBADBADBAD": "noSuchVault"
```

Comment:
Add a vault, which exists and is not owned by the test user.

Reply:
Do you mean _not accessible_?

Reply:
Technically there is nothing wrong if you add access tokens for yourself, even if lack permissions to access the vault.

Since @SailReal and I decided to make this part of the API a permanent one that might be useful in other cases in which a user might want to prepare key ownership before being granted access, I plan to keep it this way.